packcheck.sh

```diff
@@ -1039,24 +1039,24 @@ find_in_parents () {
 
 ensure_cabal_project() {
   CABALCMD=cabal
-  SDIST_CABALCMD=$CABALCMD
 
   if test -n "$CABAL_PROJECT"
   then
     require_file "$CABAL_PROJECT"
-    SDIST_CABALCMD="$CABALCMD --project-file=$CABAL_PROJECT"
     echo "Using user specified cabal project file at [$CABAL_PROJECT]"
     if test -n "$DISABLE_SDIST_BUILD"
     then # regular build
-      CABALCMD="$SDIST_CABALCMD"
+      SDIST_CABALCMD="$CABALCMD --project-file=$CABAL_PROJECT"
     else # sdist build
       # We run the command from .packcheck/<package> dir after unpacking
       # sdist
-      CABALCMD="$CABALCMD --project-file=../../$CABAL_PROJECT"
+      SDIST_CABALCMD="$CABALCMD --project-file=../../$CABAL_PROJECT"
```

Comment:
Same as the above comment, there's also a warning below.

Reply:
Answered above.